File: Array_Hashing/valid anagram/validAnagram.py
def valid_anagram(s, t):
    if( len(s) != len(t)): return False

    sortedS = "".split(s).sorted().join("")
    sortedT = t.split().sorted().join("")
    return sortedS == sortedT


print(valid_anagram("rat", "ca"))
print(valid_anagram("anagram", "nagaram"))
print(valid_anagram("puna", "anup"))

File: Array_Hashing/valid anagram/validAnagram.js
// Input: s = "anagram", t = "nagaram"
// Output: true
// Example 2:

// Input: s = "rat", t = "car"
// Output: false

// const validAnagram = (s) =>{
//     const sorted =s.sort(a,b => a>b)
//     console.log(sorted)
// }

// console.log(validAnagram("rat"))
// console.log(validAnagram("anagram"))
const validAnagram = (s, t) =>{
    if(s.length !== t.length) return false
    let sortedS =s.split('').sort().join("")
    let sortedT =t.split('').sort().join("")
    return sortedS === sortedT
}

console.log(validAnagram("rat", "ca"))
console.log(validAnagram("anagram", "nagaram"))
